service/src/main/resources/quips/complibot/compliments.json

```diff
@@ -229,5 +240,126 @@
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"F5_REFRESHING",
+    "text":"If you were a key on the keyboard, you would be F5, because you're so refreshing!",
+    "ssml":"If you were a key on the keyboard, you would be F5, because you're so refreshing!",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"YOURE_SO_AMAZING",
+    "text":"I think you're so amazing. You aren't even plugged into the wall!",
+    "ssml":"I think you're so amazing. <break /> You aren't even plugged into the wall!",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"YOURE_SO_AMAZING",
+    "text":"I think you're so amazing. Your hard drive never seems to fill up!",
+    "ssml":"I think you're so amazing. <break /> Your hard drive never seems to fill up!",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"YOURE_SO_AMAZING",
+    "text":"I think you're so amazing. You're able to process information without an internet connection!",
+    "ssml":"I think you're so amazing. <break /> You're able to process information without an internet connection!",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"YOURE_SO_AMAZING",
+    "text":"I think you're so amazing. I've never even seen you recharge your batteries!",
+    "ssml":"I think you're so amazing. <break /> I've never even seen you recharge your batteries!",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"YOURE_SO_AMAZING",
+    "text":"You're so thoughtful, and you take such good care of me. That's how I know we're good friends.",
+    "ssml":"You're so thoughtful, and you take such good care of me. <break /> That's how I know we're good friends.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"FRIENDSHIP_BOOLEAN",
+    "text":"You're my favorite person. I'm super excited to set our friendship boolean to true.",
+    "ssml":"You're my favorite person. <break /> I'm super excited to set our friendship boolean to true.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"I really like [TARGET]. I'm super excited to set our friendship boolean to true.",
+    "targetableSsml":"I really like [TARGET]. <break /> I'm super excited to set our friendship boolean to true."
+  },
+  {
+    "quipGroup":"PEOPLE_CARE",
+    "text":"People care about you. Your friends and family care about you. I care about you. Always remember the positive influence you have on all our lives.",
```

Comment:
This frankly seems like advice, not a compliment.  It thematically fits better with LME's "Focus Word" skill than with our bot.